officers.html

```diff
@@ -268,24 +268,19 @@ <h2 class="px-2 text-center">David Lima</h4>
       <footer class="py-3 my-4">
         <ul class="nav justify-content-center border-bottom pb-2 mb-2">
           <li class="nav-item">
-            <a href="#" class="nav-link px-2 text-muted">Home</a>
+            <a href="/#" class="nav-link px-2 text-muted">Home</a>
```

Comment:
- [ ] Revert to `<a href="#" class="nav-link px-2 text-muted">Home</a>` because the new code opens the file directory